### DISCUSSION TITLE
How set defaultValue property in CheckPicker after fetched data from RestAPI?

### DISCUSSION THREAD

**Discussion by viweei** [rsuite/rsuite#2764] [Q&A]
I want set default checked in CheckPicker after fetched data form RestAPI. but  defaultValue property is Uncontrolled Components.
have you any good idea?

**Comment by SevenOutman** [comment]
@viweei You may want to not render the CheckPicker component until the data is retrieved.

```jsx
{isLoaded ? (
  <CheckPicker data={data} defaultValue={selectedValues} />
) : null}
```
